src/app/api/user/profile/route.ts

```
@@ -0,0 +1,329 @@
+
+    switch (action) {
+      case 'update_preferences':
+        const { data: updatedProfile, error: updateError } = await supabase
+          .from('profiles')
+          .update({ 
+            preferences: data.preferences,
+            updated_at: new Date().toISOString()
+          })
+          .eq('id', user.id)
+          .select()
+          .single()
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)